open-metadata-implementation/adapters/open-connectors/configuration-store-connectors/configuration-encrypted-file-store-connector/src/main/java/org/odpi/openmetadata/adapters/adminservices/configurationstore/encryptedfile/DocStoreErrorCode.java

```diff
@@ -78,6 +78,10 @@
             "Unable to delete the encrypted configuration file {0}; exception was {1} with message {2}.",
             "The system was unable to delete the encrypted configuration file.",
             "Review the full stack trace in the logs to troubleshoot further. Then retry the request."),
+    CONFIG_RETRIEVE_ALL_ERROR(400, "ENCRYPTED-DOC-STORE-400-014 ",
+                        "Unable to retrieve the encrypted configuration files; exception was {0} with message {1}.",
```

Comment:
Would be useful to include the name of the directory/file it was trying to access.

Reply:
added